app/queries/email_archive_query.rb

```diff
@@ -19,7 +19,6 @@ def fields
       message_ids,
       :finished_sending_at,
       :id,
-      :marked_as_spam,
```

Comment:
We should also mark this column as ignored in the model so that it can be safely removed. For example: https://github.com/alphagov/email-alert-api/pull/1348

Otherwise when you run the migration in https://github.com/alphagov/email-alert-api/pull/1353 there's a good chance the application will start erroring until the processes restart.

Reply:
Ah yes, good catch, thanks